### DISCUSSION TITLE
Client syslog empty

### DISCUSSION THREAD

**Discussion by mecrider** [ltsp/ltsp#293] [Q&A]
Where do clients send their system log entries by default? I was attempting to troubleshoot a problem with an application on Ubuntu 20.04, and looked in /var/log/syslog where that application typically puts its logs, but it is 0 bytes.

**Comment by alkisg** [comment]
Logging nowadays is managed by journalctl. You can run `journalctl -f` to follow it dynamically, or `journalctl -u unit` to see the output of a specific unit, etc etc.

Debian-based distributions duplicate that logging to /var/log, which is a waste of RAM on netbooted clients, so LTSP disables rsyslog by default. Hopefully Debian will disable the duplicate logging in a subsequent release.

One way to re-enable rsyslog would be to put the following in ltsp.conf:

```shell
[clients]
PRE_INIT_KEEP_RSYSLOG="sed 's/rsyslog.service//' -i /usr/share/ltsp/client/init/56-mask-services.sh"
```

**Comment by mecrider** [comment]
Thanks for the info. journalctl works very well for my situation after giving root a password (per issue ltsp/ltsp#286).
